### PR TITLE
Adding fields for SRGB and AdjSID

### PR DESCRIPTION
**Pull request by rszarecki**

### CODE REVIEW

**Comment by jnpr-community-netdev** [comment]
Can one of the admins verify this patch?

**Comment by jnpr-community-netdev** [comment]
Can one of the admins verify this patch?

File: lib/jnpr/junos/op/teddb.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ TedView:
   link-in: { ted-database-link-in : int }
   link-out: { ted-database-link-out : int }
   protocol: ted-database-protocol
+  srgb_base: { ted-spring-capability/ted-spring-srgb-block/ted-spring-srgb-block-start : int }
```
Comment (rsmekala) [inline lib/jnpr/junos/op/teddb.yml]: Please change `_` or to `-` to maintain uniformity. Also, change `srgb_flag` to `srgb_flags`

File: lib/jnpr/junos/op/teddb.yml [hunk 2]
```diff
@@ -33,6 +36,10 @@ _linkView:
   localADD: ted-link-local-address
   metric: { ted-link-metric : int }
   color: admin-groups/color
+  SR_Adj_SID_label: {ted-adj-sids/ted-link-p2p-adj-sid/ted-link-p2p-adj-sid-label : int}
```
Comment (rsmekala) [inline lib/jnpr/junos/op/teddb.yml]: Same as above. Can you not use capitalized letters if possible.

**Review by rahkumar651991 — APPROVED** [review]

**Pull request merged**